spinn_utilities/make_tools/__init__.py

```diff
@@ -13,7 +13,6 @@
 # You should have received a copy of the GNU General Public License
 # along with this program.  If not, see <http://www.gnu.org/licenses/>.
 from .file_converter import FileConverter
-from .replacer import Replacer
 
 # converter NOT included here double import warning in automatic make
-__all__ = ["FileConverter", "Replacer"]
+__all__ = ["FileConverter"]
```

Comment:
Any reason replacer is gone?

Reply:
yes:
/usr/lib/python3.9/runpy.py:127: RuntimeWarning: 'spinn_utilities.make_tools.replacer' found in sys.modules after import of package 'spinn_utilities.make_tools', but prior to execution of 'spinn_utilities.make_tools.replacer'; this may result in unpredictable behaviour